rewards/utils/emission_utils.py

```diff
@@ -139,3 +144,12 @@ def parse_nft_weight_schedule(weight_schedule: List) -> NFTWeightSchedule:
 
 def parse_nft_weight_schedules(weight_schedules: List) -> List[NFTWeightSchedule]:
     return [parse_nft_weight_schedule(ws) for ws in weight_schedules]
+
+def get_across_lp_multiplier() -> float:
+    bridge = make_contract(ACROSS_BRIDGE, abi_name=Abi.BridgePoolProd, chain=Network.Ethereum)
+    
+    liquid_reserves = bridge.liquidReserves().call()
+    utilized_reserves = bridge.utilizedReserves().call()
```

Comment:
Utilized reserves can be negative, so we need to take the max of the response from this call and 0.

https://etherscan.io/address/0x43298F9f91a4545dF64748e78a2c777c580573d6#code#F1#L743
Here's an example in the solidity